plenum/test/test_node.py

```diff
@@ -379,7 +379,7 @@ def __init__(self,
         self.pluginPaths = pluginPaths
 
         self.testNodeClass = testNodeClass
-        self.nodes = OrderedDict()  # type: Dict[str, TestNode]
+        self.nodes = SortedDict()  # type: Dict[str, TestNode]
```

Comment:
It depends on the fixture you are using. `nodeSet` returns a `TestNodeSet`, which is "collection" of old style nodes, it supports indexing (by name and index) and iteration. `txnPoolNodeSet` returns a list of new style nodes. So if you were using `nodeSet`, you can simply choose last node as `nodeSet.nodes_by_rank[-1]`